Sources/_StringProcessing/RegexDSL/Core.swift

```diff
@@ -247,10 +152,4 @@ public struct EmptyCapture: EmptyCaptureProtocol {}
 extension Array: EmptyCaptureProtocol where Element: EmptyCaptureProtocol {}
 extension Optional: EmptyCaptureProtocol where Wrapped: EmptyCaptureProtocol {}
 
-public protocol MatchProtocol {
-  associatedtype Capture
-}
-extension Substring: MatchProtocol {
-  public typealias Capture = EmptyCapture
-}
```

Comment:
FWIW, #127 removes `MatchProtocol`, so you can probably leave it here so that there's one fewer conflict.